src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -277,7 +278,7 @@ public Response getDataset(@PathParam("datasetId") Integer datasetId){
             if (Objects.isNull(dataset)) {
                 throw new NotFoundException("Could not find the dataset with id: " + datasetId.toString());
             }
-            return Response.ok(dataset, MediaType.APPLICATION_JSON).build();
+            return Response.ok(unmarshal(dataset)).build();
```

Comment:
forced using Gson by calling `unmarshal` on all endpoints which return datasets. I manually checked against develop and it appears to send the exact same object back.